src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,4 +60,9 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+	@Override
+	public String getPrintableString(Printable... printables) {
```

Comment:
You don't need to use `Printable... printables` here. Recollect from the discussion.